### ISSUE TITLE
Question about search quality

### ISSUE THREAD

**Issue by groue** [CocoaPods/cocoapods.org#294]
Hello,

The [search for Swift pods about SQLite](https://cocoapods.org/?q=lang%3Aswift%20sqlite) only shows pods that contain SQLite in their *name*. The consequence is that pods like [GRDB.swift](https://cocoapods.org/pods/GRDB.swift) (94 quality points) are not present in the search results. And they are not discoverable unless you already know their name.

I don't understand. Weren't quality points and https://twitter.com/cremedelapods introduced to favor pods discoverability and foster good quality pods? Shouldn't the search be part of it? What do you think?

**Comment by orta** [comment]
[Name](https://github.com/CocoaPods/search.cocoapods.org/blob/master/lib/search.rb#L84) takes precedence [over summary](https://github.com/CocoaPods/search.cocoapods.org/blob/master/lib/search.rb#L120) in our search  which then chunks by found type and ordered by quality. As there are so many with SQLite in their name, then they show first. I'd assume GRDB would be high on per-summary or description, but not on per name I'm afraid.

**Comment by groue** [comment]
@orta OK I guess you already had a hard time figuring out the best relative weights of textual fields - even if this sometimes creates something similar to cybersquatting (or at least an absurd bonus to "well-named" pods).

Still, I don't know where to reach pods that would eventually be matched according to their summary or description in the following screenshot (Safari 9.1.2).

<img width="812" alt="capture d ecran 2016-09-20 a 12 16 38" src="https://cloud.githubusercontent.com/assets/54219/18666426/1e58a1d6-7f2c-11e6-88f0-8e34bd4f81da.png">

**Comment by orta** [comment]
I wonder if my recent work de-duplication #289  has broken any grouping other than name - I'm not seeing anything for queries related to anything other than name

**Comment by floere** [comment]
@orta I can't check it right now, but did you look at the JSON response?

On Di., 20. Sep. 2016 at 12:31 Orta

<
mailto:Orta <notifications@github.com>
> wrote:

a, pre, code, a:link, body { word-wrap: break-word !important; }

I wonder if my recent work de-duplication
https://github.com/CocoaPods/cocoapods.org/pull/289
has broken any grouping other than name - I'm not seeing anything for queries related to anything other than name

—

You are receiving this because you are subscribed to this thread.

Reply to this email directly,
https://github.com/CocoaPods/cocoapods.org/issues/294#issuecomment-248263791
, or

**Comment by floere** [comment]
@orta Indeed, the results are provided by the search backend, but they may be removed by the deduplication.

**Comment by orta** [comment]
De-duplication isn't working, #289 should be reverted 
![screen shot 2016-10-04 at 19 10 01](https://cloud.githubusercontent.com/assets/49038/19086491/4fecbfbc-8a66-11e6-9820-5d9ee0ab15e9.png)

**Comment by orta** [comment]
![screen shot 2016-10-04 at 19 12 15](https://cloud.githubusercontent.com/assets/49038/19086541/7b3eb4c2-8a66-11e6-9b28-992a0bbecf27.png)
